spec/schemas/_core.msearch.yaml

```diff
@@ -119,44 +119,44 @@ components:
           $ref: '_core.search.yaml#/components/schemas/SourceConfig'
         fields:
           description: |-
-            Array of wildcard (*) patterns. The request returns values for field names
-            matching these patterns in the hits.fields property of the response.
+            An array of wildcard (*) patterns. The request returns values for field names
+            matching these patterns in the `hits.fields` property of the response.
           type: array
           items:
             $ref: '_common.query_dsl.yaml#/components/schemas/FieldAndFormat'
         terminate_after:
           description: |-
-            Maximum number of documents to collect for each shard. If a query reaches this
-            limit, OpenSearch terminates the query early. OpenSearch collects documents
-            before sorting. Defaults to 0, which does not terminate query execution early.
+            The maximum number of documents to collect for each shard. If a query reaches this
+            limit, OpenSearch stops the query early. OpenSearch collects documents
+            before sorting. Default is 0, which does not terminate query execution early.
           type: number
         stats:
           description: |-
-            Stats groups to associate with the search. Each group maintains a statistics
+            The statistics groups to associate with the search. Each group maintains a statistics
             aggregation for its associated searches. You can retrieve these stats using
-            the indices stats API.
+            the Index Stats API.
           type: array
           items:
             type: string
         timeout:
           description: |-
             Specifies the period of time to wait for a response from each shard. If no response
             is received before the timeout expires, the request fails and returns an error.
-            Defaults to no timeout.
+            Default is no timeout.
           type: string
         track_scores:
-          description: If true, calculate and return document scores, even if the scores are not used for sorting.
+          description: When true, calculates and returns any document scores, even if the scores are not used for sorting.
           type: boolean
         track_total_hits:
           $ref: '_core.search.yaml#/components/schemas/TrackHits'
         version:
-          description: If true, returns document version as part of a hit.
+          description: When true, returns the document version as part of a hit.
           type: boolean
         runtime_mappings:
           $ref: '_common.mapping.yaml#/components/schemas/RuntimeFields'
         seq_no_primary_term:
           description: |-
-            If true, returns sequence number and primary term of the last modification
+            When true, returns sequence number and primary term of the last modification
             of each hit. See Optimistic concurrency control.
```

Comment:
Should there be a link here?

Reply:
@dblock or @nhtruong: Do we know what Optimistic concurrency control is referring to? Can find any reference to this in the documentation?